### PR TITLE
rename default dir  daal->dal

### PR DESCRIPTION
**Pull request by napetrov**
# Description
rename default dir daal -> dal
create link from daal -> dal

### CODE REVIEW

File: makefile [hunk 1]
```diff
@@ -1022,6 +1022,19 @@ ifeq ($(OS_is_win),yes)
...
+# Note: The `ln` command does not support `-r` on MacOS, so we
+#       `cd`to the lib directory first and then create symlink.
+ifeq ($(if $(or $(OS_is_lnx),$(OS_is_mac)),yes,),yes)
+_release_c: 
+  cd $(RELEASEDIR) && ln -sf daal dal
+endef
+
+ifeq ($(OS_is_win),yes)
+_release_c: 
+  cd $(RELEASEDIR) && mklink /D daal dal
```
Comment (rlnx) [inline makefile:1035]: Does it work without admin access?

Comment (napetrov) [reply makefile:1035]: yes. as we create link in some user directory and it's sufficient to have use right there

File: makefile [hunk 2]
```diff
@@ -1022,6 +1022,21 @@ ifeq ($(OS_is_win),yes)
...
 endif
 
+# Adds symlink to the old library directory
+# Note: The `ln` command does not support `-r` on MacOS, so we
+#       `cd`to the lib directory first and then create symlink.
+ifeq ($(if $(or $(OS_is_lnx),$(OS_is_mac)),yes,),yes)
+_release_c:
+	cd $(RELEASEDIR) && ln -sf dal daal
+
+endef
```
Comment (KulikovNikita) [inline makefile:1032]:
```suggestion
endif
```
Misprint?

Comment (napetrov) [reply makefile]: Yes - thanks!

**Pull request closed without merging**